clusterloader2/testing/nichotr/config.yaml

```diff
@@ -0,0 +1,143 @@
+#Constants
+{{$NODE_MODE := DefaultParam .NODE_MODE "allnodes"}}
+{{$NODES_PER_NAMESPACE := DefaultParam .NODES_PER_NAMESPACE 1}}
+{{$PODS_PER_NODE := DefaultParam .PODS_PER_NODE 56}}
+{{$ENABLE_CHAOSMONKEY := false}}
+
+#Pod Resource Request
+{{$POD_CPU := DefaultParam .POD_CPU 20}}
```

Comment:
I'm guessing these are in millicpu and megabytes. Consider POD_CPU_MILLI and POD_MEMORY_MB to make this clear.

We should parameterize these based off of # nodes and node size.
We probably need
.Nodes
.NodeCPUMillis
.NodeMemoryMb

and then we can compute 
PodCPUMillis = ( (NodeCPUMillis - SystemPodCPUMillis) / PodsPerNode )

We can do the math, but we need to figure out how much CPU the node and daemonsets are using. In my cluster it's  KubeProxy (100m) and AWSNodeCNI (10m).

Reply:
Line 77 and 78 have those units, but this on the TODO list for sure.